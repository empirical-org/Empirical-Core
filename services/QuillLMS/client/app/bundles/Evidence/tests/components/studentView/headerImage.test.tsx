import { shallow } from 'enzyme';
import * as React from 'react';

import HeaderImage from '../../../components/studentView/headerImage';

const mockProps = {
  headerImage: <img alt="header" />,
  passage: {
    image_caption: 'test caption',
    image_attribution: 'test attribution'
  }
};

describe('Header Image component', () => {
  const container = shallow(<HeaderImage {...mockProps} />);

  it('should render Header Image', () => {
    expect(container).toMatchSnapshot();
  });
});
