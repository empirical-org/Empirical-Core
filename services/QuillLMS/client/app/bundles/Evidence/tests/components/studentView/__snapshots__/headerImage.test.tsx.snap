// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Image component should render Header Image 1`] = `
<Fragment>
  <section
    className="header-image-container"
  >
    <img
      alt="header"
    />
    <section
      className="header-image-information"
    >
      <p
        className="header-image-caption"
      />
      <Tooltip
        handleClick={[Function]}
        isTabbable={false}
        tooltipText="<p>test attribution</p>
"
        tooltipTriggerText="Image credit"
        tooltipTriggerTextClass="image-attribution-tooltip"
      />
    </section>
  </section>
</Fragment>
`;
